app/src/components/alert/useSeverityStyle.ts

```diff
@@ -0,0 +1,22 @@
+import { SeverityLevel } from "../types";
+
+import { dangerCSS, infoCSS, successCSS, warningCSS } from "./styles";
+
+export function useSeverityStyle(severity: SeverityLevel) {
```

Comment:
this doesn't use any react hooks so the `use` nomenclature is a bit misleading imo